### PR TITLE
Documented target option to revert to es5 syntax

### PR DESCRIPTION
**Pull request by rramaa**
`output.ecmaVersion` is removed in the release candidate
Documented the target option, through which the same thing can be achieved

### CODE REVIEW

File: src/content/migrate/5.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Deprecation warnings. You might get a lot of deprecation warnings. This is not a
 
 #### Turn off ES2015 syntax in runtime code, if necessary
 
-By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `output.ecmaVersion: 5` to revert to ES5 syntax.
+By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `config.target: ["web", "es5"]` to revert to ES5 syntax(web if target environment is browser).
```
Comment (snitin315) [inline src/content/migrate/5.md:149]:
```suggestion
By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `target: ["web", "es5"]` to revert to ES5 syntax (`web` if target environment is browser).
```

**Comment by rramaa** [comment]
Some problem with the `licence/cla` check
I have contributed before @sokra

**Comment by evilebottnawi** [comment]
Fixed

**Comment by rramaa** [comment]
The build is failing
Seems some loaders are not found
Was working before
Need to update with master?

**Review by snitin315 — APPROVED** [review]

File: src/content/migrate/5.md [hunk 2]
```diff
@@ -146,7 +147,7 @@ Deprecation warnings. You might get a lot of deprecation warnings. This is not a
 
 #### Turn off ES2015 syntax in runtime code, if necessary
 
-By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `output.ecmaVersion: 5` to revert to ES5 syntax.
+By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `target: ["web", "es5"]` to revert to ES5 syntax (`web` if target environment is browser).
```
Comment (EugeneHlushko) [inline src/content/migrate/5.md:150]:
```suggestion
By default, webpack's runtime code uses ES2015 syntax to build smaller bundles. If your build targets environments that don't support this syntax (like IE11), you'll need to set `target: ['web', 'es5']` to revert to ES5 syntax (`web` if target environment is browser).
```

**Review by EugeneHlushko — CHANGES_REQUESTED** [review]

File: src/content/migrate/5.md (hunk 2 above)
Comment (EugeneHlushko) [reply src/content/migrate/5.md:150]: We are using single quotes in JS snippets in webpack.js.org, changing for consistency

**Comment by EugeneHlushko** [comment]
Is this in-line with #4016 ?
CC @smelukov

**Comment by sokra** [comment]
Thanks

**Pull request merged**